tests/acceptance/async_adapter_tests.py

```diff
@@ -162,7 +163,30 @@ def on_exchange_declared(self, frame):
 
     def on_bad_result(self, frame):
         self.channel.exchange_delete(self.name)
-        raise AssertionError("Should not have received a Queue.DeclareOk")
+        raise AssertionError("Should not have received an Exchange.DeclareOk")
+
+
+class TestPassiveExchangeDeclareWithConcurrentClose(AsyncTestCase, AsyncAdapters):
```

Comment:
Was this test reproducing the failure prior to your fix?

Reply:
Sure does, I'll add a comment about it to clarify, since it doesn't really look like it tests anything.